### ISSUE TITLE
Vulnerability review.

### ISSUE THREAD

**Issue by kfogel** [redcross/arcdata#119]
Any vulnerabilities currently displayed by the [GitHub automagic vulnerability detection system](https://help.github.com/articles/about-security-alerts-for-vulnerable-dependencies) should be examined and handled if necessary.

(And we need to do an independent vulnerability review soon after that, but for the moment let's use the information right in front of us courtesy of GitHub.)

@pjsier and @pmacaluso3, you should both be seeing the vulnerability report when you are logged into GitHub and go to the front page of this repository.  Let me know if you have any trouble seeing it.

**Comment by pjsier** [comment]
Updating based on conversations on the call today:

Many of the initial vulnerabilities were fixed by merging #127, but there are still a few left, most of which involve upgrading to Rails 4.2 which introduces challenges described in #126. For now we should upgrade `nokogiri` and `rest_api` if possible, and after that we can reduce the priority of this issue

**Comment by bnazari** [comment]
Nokogiri is heavily used in the syntonization with Volunteer Connection. We definitely need to test this prior to rolling out a patch as any interruptions would effect system functionality.

**Comment by kfogel** [comment]
@bnazari thanks for noticing that.  @pjsier and @pmacaluso3, let's isolate the `nokogiri` change to its own commit, deployed separately from any other upgrades, so that we can deploy it cautiously and roll back right away if needed.

**Comment by kfogel** [comment]
How's progress on this issue, @pjsier and @pmacaluso3?  We're getting another vulnerability warning from GitHub now, I'm sure you've noticed (let me know if you can't see it when you log in to GitHub and view this repository).

**Comment by pjsier** [comment]
@kfogel I've got #139 open for nokogiri, and I was waiting to merge and deploy that until we finish #120. We can do that sooner though if that works.

I can open up another PR for the new warning, and then it was my impression that we're lowering the priority of this issue. The other vulnerabilities all relate more to #126 and moving to Rails 4.2

**Comment by pjsier** [comment]
Paperclip is one dependency we'll need to update, but it's blocked by #126 as well. Flagging it here for now
